### PR TITLE
Spring Data Home Work

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.1.1</version>
+                <configuration>
+                    <finalName>homeworks</finalName>
+                </configuration>
+            </plugin>
+            <plugin>
```
Comment (SergeyPasko) [inline pom.xml]: Why do you use this plugin?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    </build>
+
+    <dependencies>
+
+        <!--Testing-->
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
```
Comment (SergeyPasko) [inline pom.xml]: if you use version from properties - use this appropriach for all dependensies

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SergeyPasko) [inline pom.xml]: Drop all not need dependensies!!!!!

File: src/main/java/dataBase/entity/Office.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        buffer.append("region").append("='").append(getRegion()).append("' ");
+        buffer.append("target").append("='").append(getTarget()).append("' ");
+        buffer.append("sales").append("='").append(getSales()).append("' ");
+        buffer.append("]");
+
+        return buffer.toString();
+    }
+
+    /*
+    StringBuffer buffer = new StringBuffer();
```
Comment (SergeyPasko) [inline src/main/java/dataBase/entity/Office.java]: ????

File: src/main/java/springData/service/OfficesServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        LOG.debug("getAllOffices");
+        HashSet<Office> result = new HashSet<>(officesRepository.findAll());
+        LOG.debug("getAllOffices return {} records", result.size());
+        return result;
+    }
+
+    @Override
+    public Office getOfficeByCity(String city) {
+        LOG.debug("getOfficeByCity, city={}", city);
+        Office result = officesRepository.findById(city).orElse(null);
```
Comment (SergeyPasko) [inline src/main/java/springData/service/OfficesServiceImpl.java]: City it isn't ID!!!! It is critical mistake. ID in your table another column

File: src/main/resources/log4j2.xml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+        <SMTP name="FEMAIL"
+              SMTPProtocol="smtps"
+              SMTPHost="smtp.gmail.com"
+              SMTPPort="465"
+              SMTPUsername="lifeguardtrashbox@gmail.com"
+              SMTPPassword="rereheprf03"
+              To="lifeguardtrashbox@gmail.com"
+              from="lifeguardtrashbox@gmail.com"
+              Subject="Calculator Error Log">
```
Comment (SergeyPasko) [inline src/main/resources/log4j2.xml]: What name???

File: pom.xml (hunk 2 above)
Comment (Artsapollo) [reply pom.xml]: didn't get you

Comment (Artsapollo) [reply pom.xml]: Didn't get you

File: src/main/resources/log4j2.xml (hunk 6 above)
Comment (Artsapollo) [reply src/main/resources/log4j2.xml]: About that also

File: pom.xml (hunk 2 above)
Comment (SergeyPasko) [reply pom.xml]: Якщо пишеш версії ліб через проперті мавена то роби це для всіх ліб а не для частини

File: src/main/resources/log4j2.xml [hunk 7]
```diff
@@ -24,15 +24,15 @@
...
               SMTPProtocol="smtps"
               SMTPHost="smtp.gmail.com"
               SMTPPort="465"
               SMTPUsername="lifeguardtrashbox@gmail.com"
-              SMTPPassword="rereheprf03"
+              SMTPPassword="xxxx"
               To="lifeguardtrashbox@gmail.com"
               from="lifeguardtrashbox@gmail.com"
-              Subject="Calculator Error Log">
+              Subject="Log Error">
```
Comment (SergeyPasko) [inline src/main/resources/log4j2.xml]: I wrote only Subject about

**Pull request merged**